rpc/grpc/core/proto/rpc.proto

```diff
@@ -851,3 +851,19 @@ message GetDaaScoreTimestampEstimateResponseMessage{
         repeated uint64 timestamps = 1;
         RPCError error = 1000;
 }
+
+message GetPriorityFeeEstimateRequestMessage {}
+
+message GetPriorityFeeEstimateResponseMessage {
+  oneof fee_per_mass {
+    Virtual virtual = 2; // 1 is reserved for `All`
+  }
+  uint64 mempool_total_mass = 11;
```

Comment:
What would be the usage of total mass from the client side? I'm not sure we want to commit ourselves to such an API. I have another suggestion for a more abstracted response

Reply:
based on this value we can track both: absolute values/relative to bps and block limit. that's more flexible

Reply:
Wdyt @aspect?